src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/outertypefilename/Example2.java

```diff
@@ -0,0 +1,15 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="OuterTypeFilename"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.outertypefilename;
+
+// xdoc section -- start
+class Foo2 { // violation, 'The name of the outer type and the file do not match.'
+
+}
+class Example2ButNotSameName {}
+// xdoc section -- end
```

Comment:
```
// xdoc section -- start
class Example2ButNotSameNameAsFile {}  // violation, 'The name of the outer type and the file do not match.'
// xdoc section -- end
```

Reply:
Done....
also removed the empty block space for all other example files like you suggested in my other PR, hope its all good.

Reply:
Please update all other examples to use this pattern.
And please update text above example , see generated website page, there should be no Foo anymore

Reply:
Made the changes..
Please review if it looks good now